php-cs-fixer

```diff
@@ -54,8 +54,14 @@ try {
     }
 }
 
+use Composer\XdebugHandler\XdebugHandler;
 use PhpCsFixer\Console\Application;
 
+// Restart if xdebug is loaded, unless the environment variable PHP_CS_FIXER_ALLOW_XDEBUG is set.
+$xdebug = new XdebugHandler('PHP_CS_FIXER', '--ansi');
```

Comment:
just about this one,
if we provide `--ansi` on ansi-supported terminal, it pass it to our console application and it will use colours. great.

in general, `--ansi` is `<info> --ansi</info> Force ANSI output`.

while I agree that this is needed for terminals that support colours, i'm not sure how it would behave in terminal which doesn't support colours...

maybe we shall pass it only if terminal [hasColorSupport](https://github.com/symfony/console/blob/0d62adf9f4dc46a837560dee07b2b94675ddc492/Output/StreamOutput.php#L92) ?

Reply:
XdebugHandler only passes it to the sub process if the parent supports colors. All this does is tell XdebugHandler what parameter is used by this application, not that it should apply it.